src/chatkit.ts

```diff
@@ -367,6 +367,36 @@ export default class Chatkit {
     })
   }
 
+  updatePermissionsForGlobalRole(
+    roleName: string,
+    permissionsToAdd: Array<string>,
+    permissionsToRemove: Array<string>,
+  ): Promise<any> {
+    var permsToCheck: Array<string> = permissionsToadd.concat(permissionsToRemove)
+    (permsToCheck).forEach((perm) => {
+      if (validGlobalPermissions.indexOf(perm) < 0) {
+        throw new Error(`Permission value "${perm}" is invalid`);
+      }
+    })
+
+    return updatePermissionsForRole(roleName, 'global', permissionsToadd, permissionsToRemove)
+  }
+
+  updatePermissionsForRoomRole(
+    roleName: string,
+    permissionsToAdd: Array<string>,
+    permissionsToRemove: Array<string>,
+  ): Promise<any> {
+    var permsToCheck: Array<string> = permissionsToadd.concat(permissionsToRemove)
+    permsToCheck.forEach((perm) => {
+      if (validRoomPermissions.indexOf(perm) < 0) {
+        throw new Error(`Permission value "${perm}" is invalid`);
+      }
+    })
+
+    return updatePermissionsForRole(roleName, 'room', permissionsToadd, permissionsToRemove)
```

Comment:
`updatePermissionsForRole` -> `this.updatePermissionsForRole`

`permissionsToadd` -> `permissionsToAdd`